### PR TITLE
spliting keepalived and haproxy in 2 jobs

### PR DESCRIPTION
**Pull request by axelfauvel**
/CC @gberche-orange @poblin-orange

### CODE REVIEW

File: jobs/keepalived/spec [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: keepalived
+
+description: "The keepalived job can be used to add a VRRP IP address to enforce HA on haproxy release"
+
```
Comment (gberche-orange) [inline jobs/keepalived/spec:5]: Should we start documenting the other use cases for VRRP protocol such as controlling an AWS elastic ip ?

The keepalived job can be used to add a VRRP IP address to enforce HA on haproxy release, **or simply to control an external SDN system through use of custom control notification scripts**

**Review by gberche-orange — COMMENTED** [review]
Thanks @axelfauvel for this split.

In the future, we might need to start differentating the 2 use-cases for the VRRP:
1- control the LVS for exposing a VIP (through emitting the gratuition ARPs)
2- control a distinct external SDN system such as an AWS ElasticIP (through custom VRRP failover notification scripts) 

Suggestions to record for a future change:
- mention the different use cases in the documentation
- start making the comments at https://github.com/orange-cloudfoundry/haproxy-boshrelease/blob/keepalive/jobs/haproxy/templates/keepalived.config.erb#L5 more generic and less dependent on being collocated with haproxy

File: jobs/keepalived/spec (hunk 1 above)
Comment (axelfauvel) [reply jobs/keepalived/spec:5]: Maybe we should wait for this feature to be implemented before mentioning it in spec file ?

**Comment by axelfauvel** [comment]
@gberche-orange : I think your last sentence is incomplete :)

start making the comments at https://github.com/orange-cloudfoundry/haproxy-boshrelease/blob/keepalive/jobs/haproxy/templates/keepalived.config.erb#L5 more

**Comment by gberche-orange** [comment]
sorry

> - start making the comments at https://github.com/orange-cloudfoundry/haproxy-boshrelease/blob/keepalive/jobs/haproxy/templates/keepalived.config.erb#L5 more generic and less dependent on being collocated with haproxy

File: jobs/keepalived/spec (hunk 1 above)
Comment (gberche-orange) [reply jobs/keepalived/spec:5]: agreed, good point

File: jobs/keepalived/spec [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+templates:
+  keepalived_ctl: 		bin/keepalived_ctl
+  keepalived.config.erb: 	config/keepalived.config
+
+properties:
+  keepalived.vip:
+    description: Virtual IP V4 address that will be given to master 
+  keepalived.healthcheck_name:
+    description: name of healthcheck that will be performed
```
Comment (gberche-orange) [inline jobs/keepalived/spec]: label displayed for the health check. Will appear in keepalive traces e.g. "Keepalived_vrrp[4558]: VRRP_Script(check_haproxy) failed"

**Pull request merged**